python/lsst/ts/IntegrationTests/script_queue_controller.py

```diff
@@ -92,16 +92,19 @@ async def do_add(self, data: tuple) -> None:
         await self.evt_script.set_write(
             scriptSalIndex=len(self.queue_list),
             processState=ScriptProcessState.UNKNOWN,
+            scriptState=0,  # UNKNOWN
             force_output=True,
         )
         await self.evt_script.set_write(
             scriptSalIndex=len(self.queue_list),
             processState=ScriptProcessState.LOADING,
+            scriptState=1,  # UNCONFIGURED
```

Comment:
I think the Script state enums would be better than comments. What happens if the enums change values?